### PR TITLE
MGMT-6548 - Fix kubeconfig is missing for pytest jobs

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

**Comment by YuviGold** [comment]
We need to find a solution how the naming of the kubeconfig would fit for openshift-tests expected KUBECONFIG path.

File: Makefile [hunk 1]
```diff
@@ -81,7 +81,7 @@ IMAGE_NAME=test-infra
 IMAGE_REG_NAME=quay.io/itsoiref/$(IMAGE_NAME)
 
 # oc deploy
-KUBECONFIG := $(or $(KUBECONFIG),${HOME}/.kube/config)
+#KUBECONFIG := $(or $(KUBECONFIG),${HOME}/.kube/config)
```
Comment (YuviGold) [inline Makefile:84]: ?

File: discovery-infra/tests/conftest.py [hunk 2]
```diff
@@ -2,19 +2,48 @@
...
 from test_infra.utils import operators_utils
 
 
-def _get_cluster_name():
-    cluster_name = utils.get_env('CLUSTER_NAME', f'{consts.CLUSTER_PREFIX}')
-    if cluster_name == consts.CLUSTER_PREFIX:
-        cluster_name = cluster_name + '-' + str(uuid.uuid4())[:consts.SUFFIX_LENGTH]
-    return cluster_name
+@dataclass
+class ClusterName:
```
Comment (YuviGold) [inline discovery-infra/tests/conftest.py:16]: please rebase after the clustername PR would be merged

File: discovery-infra/tests/conftest.py [hunk 3]
```diff
@@ -2,19 +2,48 @@
...
+        if self.prefix == consts.CLUSTER_PREFIX:
+            return self.prefix + '-' + self.suffix
+        return self.prefix
+
+
+cluster_name = ClusterName()
+
+
+def get_kubeconfig_path():
+    kubeconfig_dir = Path("/tmp/kubeconfig")
```
Comment (YuviGold) [inline discovery-infra/tests/conftest.py:39]: why `/tmp` and not `/build` directory like we do in the old workflow?

File: Makefile (hunk 1 above)
Comment (eliorerz) [reply Makefile:89]: It defined in the code, no need to set it also here

**Review by YuviGold — APPROVED** [review]

**Pull request merged**